script.js

```diff
@@ -38,9 +40,21 @@
 // ====================================================================
 
 
+var ordinalNum = parseInt(prompt("Provide a number", "0"));
+var NumSplit = ordinalNum.split("");
```

Comment:
Line 44 throws an error. The .split() method works on strings, but not integers, while in line 43, you've made sure that ordinalNum is an integer...

Reply:
Would it be better to just prompt the user input as a string in order to split it better